### ISSUE TITLE
Mod 4 year range checks

### ISSUE THREAD

**Issue by m-j-horner** [episphere/questionnaire#424]
@huip-ims @joshid-ims 
I reviewed the YEAR range checks in Module 4. These are correct and no further programming revisions are required for the checks. 

Module 4 does not contain age range checks.

**Comment by joshid-ims** [comment]
@m-j-horner, Module 4 still has RcrtUP_YOB_v1r0 which needs to be replaced with age, right?

**Comment by m-j-horner** [comment]
@joshid-ims  no, because the questions in module 4 are asking about the YEAR a ppt moved into or out of their residence, not their age. 

![image](https://github.com/user-attachments/assets/e08c08c1-10ef-497b-a036-cb5a4309e033)

**Comment by joshid-ims** [comment]
Ok, thanks!

**Comment by woodruffr** [comment]
We've been told that the RcrtUP_YOB_v1r0 is treated like "age" in the renderer. When we've tried to use it, we are getting an error. What is the correct form to add into Previous Results to test this code in the renderer?

![image (1)](https://github.com/user-attachments/assets/1ac2deeb-92dc-49ee-b1bb-7057cd37f734)
@anthonypetersen, what is the format for this in the renderer?

**Comment by woodruffr** [comment]
I note that is M1, which generally uses Quest IDs.

**Comment by JoeArmani** [comment]
> We've been told that the RcrtUP_YOB_v1r0 is treated like "age" in the renderer. When we've tried to use it, we are getting an error. What is the correct form to add into Previous Results to test this code in the renderer?
> 
> ![image (1)](https://private-user-images.githubusercontent.com/81166095/381293049-1ac2deeb-92dc-49ee-b1bb-7057cd37f734.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzMTAzODcsIm5iZiI6MTczMDMxMDA4NywicGF0aCI6Ii84MTE2NjA5NS8zODEyOTMwNDktMWFjMmRlZWItOTJkYy00OWVlLWIxYmItNzA1N2NkMzdmNzM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDE3NDEyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdlMjYyY2I3OGMwMzBkZjNhMGNmYmMwM2IzNzcwM2ZkZTBmMGFlMTk0MDA2YjNkYjI0MjQyMmVjZWM4OGM2ZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ubiX7NzfDMjjMsDrY7mOPpdj6Mb7J3l49MmdZVg-f-w) @anthonypetersen, what is the format for this in the renderer?

@woodruffr @cunnaneaq 
This should work for `RCTRUP_YOB_V1R0`

```json
{
    "age": "66",
    "AGE": "66",
    "firstName": "John",
    "RCRTUP_YOB_V1R0": 1970
}
```
<img width="678" alt="Screenshot 2024-10-30 at 1 50 08 PM" src="https://github.com/user-attachments/assets/53a96d1b-41fb-40a4-8db7-5e6054dbe5a8">

**Comment by anthonypetersen** [comment]
> We've been told that the RcrtUP_YOB_v1r0 is treated like "age" in the renderer. When we've tried to use it, we are getting an error. What is the correct form to add into Previous Results to test this code in the renderer?
> 
> ![image (1)](https://private-user-images.githubusercontent.com/81166095/381293049-1ac2deeb-92dc-49ee-b1bb-7057cd37f734.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzMTE3MzIsIm5iZiI6MTczMDMxMTQzMiwicGF0aCI6Ii84MTE2NjA5NS8zODEyOTMwNDktMWFjMmRlZWItOTJkYy00OWVlLWIxYmItNzA1N2NkMzdmNzM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDE4MDM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5ZWQ4ZjU0NWU5YzdjMWM0ZDk2NWM3ZWVkNzZiOTNhNDVhNTc0NDhiMjQzMDkxNzY1ZTJlNGZlZGZlNDI5ZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.d34y2o1exOBv8EYMc-bHam3CfyM1f9g1XENbu7QP90o) @anthonypetersen, what is the format for this in the renderer?

Where did this come from?

**Comment by woodruffr** [comment]
@anthonypetersen From Julie before she left. We were told it comes from the profile and Quest will get the information like it does with "age".

**Comment by woodruffr** [comment]
@anthonypetersen, Thank you! That worked in the renderer.

**Comment by woodruffr** [comment]
@JoeArmani, Thank you. That worked. @anthonypetersen Julie posted that info here: https://github.com/episphere/questionnaire/issues/344#issuecomment-2086945853.

**Comment by cunnaneaq** [comment]
I updated the year range checks in the module 4 English and Spanish questionnaire documents

**Comment by woodruffr** [comment]
The English M4 Questionnaire with this fix was delivered today.

**Comment by cunnaneaq** [comment]
year range checks updated in M4
